ste/sender-blobSymlinks_other.go

```diff
@@ -0,0 +1,7 @@
+// +build !linux
+
+package ste
+
+func (s *blobSymlinkSender) getExtraProperties() error {
+	return nil
```

Comment:
> why is this not doing what Linux one is doing?
This file is a generic catch-all file for OSes unsupported by the unix properties feature.
> Do you plan to implement it later?
Can you tell me what you think it would do? I don't see anything to implement-- We can't guarantee every other OS has posix properties. Long and short, no I don't plan to implement anything here. It's a no-op designed to satisfy a function call on unsupported platforms.